pybikes/smartbike.py

```diff
@@ -90,15 +90,15 @@ def __init__(self, info):
             self.extra = {
                 'uid': int(info['id']),
                 'status': info['status'],
-                'address': info['address']
+                'address': info['address'] if 'address' in info else None
             }
 
             if 'district' in info:
                 self.extra['districtCode'] = info['district']
             elif 'districtCode' in info:
                 self.extra['districtCode'] = info['districtCode']
 
-            if info['nearbyStations'] and info['nearbyStations'] != "0":
+            if 'nearbyStations' in info and info['nearbyStations'] and info['nearbyStations'] != "0":
```

Comment:
If you want to check that it's there and that it's not empty, easiest is:

```
if info.get('nearbyStations') and info['nearbyStations'] != '0':
    ...
```

of course, this is also valid

```
nearby_stations = info.get('nearbyStations')
if nearby_stations and nearby_stations != '0':
    ...
```